application/src/test/java/com/ericsson/bss/cassandra/ecchronos/application/spring/TestTomcatWebServerCustomizerPem.java

```diff
@@ -22,9 +22,10 @@
 @RunWith (SpringRunner.class)
 @SpringBootTest (webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT,
         properties = {
-                "server.ssl.certificate=./server/cert.crt",
-                "server.ssl.certificate-key=./server/key.pem",
-                "server.ssl.certificate-authorities=./server/ca.crt"
+                "server.ssl.certificate=src/test/resources/server/cert.crt",
+                "server.ssl.certificate-private-key=src/test/resources/server/key.pem",
+                "server.ssl.trust-certificate=src/test/resources/server/ca.crt",
+                "server.ssl.key-store-password=test"
```

Comment:
"server.ssl.key-store-password=test" - this shouldn't be needed, right?

Reply:
The "test" value isn't, but the "key-store-password" is. I've tested and the server doesn't start without it, giving a message "Cannot recover key". So the doc in https://docs.spring.io/spring-boot/docs/2.7.0/reference/htmlsingle/ helped, and the field was added to the documentation as required.

Reply:
Found this, https://github.com/spring-projects/spring-boot/issues/31253.
With springboot 2.7.2 this should be solved.

https://github.com/spring-projects/spring-boot/releases/tag/v2.7.2